src/main/resources/jupyter/jupyter_notebook_config.py

```diff
@@ -32,7 +32,5 @@
     'static_url_prefix':'/notebooks/' + fragment + '/static/',
     'headers': {
         'Content-Security-Policy': $(contentSecurityPolicy)
-    },
-    'websocket_ping_interval':30,
-    'websocket_ping_timeout':1800
+    }
```

Comment:
Removing this to clean up from terminal timeout experiments

Reply:
Do these settings actually affect anything user facing?

Reply:
Nope, they didn't change anything